ui/models.py

```diff
@@ -35,7 +36,10 @@ def members(self):
 
         """
         moira = utils.get_moira_client()
-        return set(moira.list_members(self.name, type=''))
+        try:
+            return set(moira.list_members(self.name, type=''))
+        except Exception as exc:
+            raise MoiraException('Something went wrong with getting moira-list members: %s', str(exc))
```

Comment:
I don't think the `%s` syntax works here, and it's not necessary to attach the two exceptions. Can you do this instead:

    except Exception as exc:
        raise MoiraException('Something went wrong with getting moira-list members') from exc

Reply:
OK